### ISSUE TITLE
Docker deployment / Docker Hub Image

### ISSUE THREAD

**Issue by 0xjjpa** [PhABC/antiScamBot_slack#20]
For easier deployment and roll-out, might be worth to have a `Dockerfile` with all the goodies included so people can easily add it through systems like `Now.sh` or other Docker cloud systems.

If no one is interested, I'll be happy to take on that task.

**Comment by PhABC** [comment]
This is a very interesting proposal. I've used docker a few times, unfortunately I haven't play enough with it to create my own container. 

Would be happy if you could help with this!

**Comment by 0xjjpa** [comment]
@PhABC Will do! Specially since [now.sh](https://now.sh) supports Docker deployment I think it's a very easy thing to try for close to zero cost.

Important to note many crypto communities are pooling some work on Rocket.chat, see https://github.com/RocketChat/Rocket.Chat/issues/8284

**Comment by PhABC** [comment]
Yeah, we are moving to rocket chat as well, but some project will still use
this bot for sometime, so trying to maintain it somewhat.

**Comment by 0xjjpa** [comment]
I'm pretty sure many communities will still use it as Slack has a very little technical entry barrier, so it's worth having it around... and since this project seems to be also used by Matrix ([antiscam](https://github.com/matrix-org/antiscam)), it's definitely worth investing time in it.

**Comment by PhABC** [comment]
Agreed!

**Comment by willianfcamara** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
